app/src/test/java/com/sdp13epfl2021/projmag/model/ImmutableProjectTest.kt

```diff
@@ -125,33 +134,20 @@ class ImmutableProjectTest {
                 assigned = get("assigned") as List<String>,
                 masterProject = get("masterProject") as Boolean,
                 bachelorProject = get("bachelorProject") as Boolean,
-                isTaken = get("isTaken") as Boolean,
                 tags = get("tags") as List<String>,
+                isTaken = get("isTaken") as Boolean,
                 description = get("description") as String,
-                id = default_id
+                allowedSections = get("allowedSections") as List<String>
             )
         }
 
         assertEquals(project, projectRebuilt)
     }
 
-    @Test
+    @Test @Ignore
```

Comment:
Is there a reason to ignore this test ?

Reply:
Thank you for taking the time to check out the test and solve the issue. This was un ignored